.github/workflows/package-manager-r-demo.yml

```diff
@@ -55,7 +55,8 @@ jobs:
           PACKAGEMANAGER_ADDRESS: ${{ matrix.package-manager.endpoint }}
           PACKAGEMANAGER_TOKEN: ${{ secrets[matrix.package-manager.token] }}
         run: |
-          curl -fOJH "Authorization: Bearer ${PACKAGEMANAGER_TOKEN}" ${PACKAGEMANAGER_ADDRESS}/__api__/download
+          os=$(R -s -e "cat(tolower(Sys.info()['sysname']))")
+          curl -f -o rspm -H "Authorization: Bearer ${PACKAGEMANAGER_TOKEN}" ${PACKAGEMANAGER_ADDRESS}/__api__/download?os=${os}
```

Comment:
I'm still looking for a better way to do this without having to manually specify the OS in the build matrix. Getting the OS in a portable way on all OSs is tricky and probably involves some more complicated `uname` output parsing. GHA's built-in `RUNNER_OS=Linux|Windows|macOS` would work well but it's not reusable outside of GHA, and still requires some translation to `linux|windows|darwin`.

In the end, I thought R's OS info was the easiest way since R is already installed and the OS names already match the expected `?os` values. But of course this only works if R is installed, so not reusable if you do need the `rspm` CLI for a Python workflow.